File: src/Level1/핸드폰번호가리기.java
package Level1;

public class 핸드폰번호가리기 {
    public String solution(String phone_number) {
        return new String(new char[phone_number.length() - 4]).replace("\0", "*") + phone_number.substring(phone_number.length() - 4, phone_number.length());
    }
}

File: src/Level1/핸드폰번호가리기.kt
fun main(){
    println(solution("01028120671"))
}

private fun solution(pn: String): String{
     var answer = ""
     for(i in pn.indices){
         answer += if(i < pn.length - 4) '*'
                 else pn[i]
     }
     return answer
}